src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -141,7 +145,9 @@ private void importCommandFromFile() {
                 ICommandImporter importer = ImporterFactory.getImporter(fileExtension);
                 String content = new String(Files.readAllBytes(Paths.get(filePath)));
                 DriverCommand command = importer.importCommands(content);
+                this.canvasManager.checkCanvas(command);
```

Comment:
This action should be performed every time command is changed. Use observer pattern (Publisher-Subscriber) implemented in the CommandManager.